authentik/blueprints/v1/tasks.py

```diff
@@ -62,29 +62,42 @@ def start_blueprint_watcher():
     if _file_watcher_started:
         return
     observer = Observer()
-    observer.schedule(BlueprintEventHandler(), CONFIG.get("blueprints_dir"), recursive=True)
+    observer.schedule(
+        BlueprintEventHandler(),
+        CONFIG.get("blueprints_dir"),
+        recursive=True,
+        event_filter=(FileCreatedEvent, FileModifiedEvent),
```

Comment:
I wasn't able to find any reference to `event_filter` in the watchdog docs

Reply:
Oh yeah, there isn't anything in the reference docs. I looked at the code directly:

https://github.com/gorakhargosh/watchdog/blob/master/src/watchdog/observers/api.py#L306
https://github.com/gorakhargosh/watchdog/blob/master/src/watchdog/observers/inotify.py#L218

Apparently it hasn't been released yet, that's maybe why it isn't in the published docs.